component/textutilities.html

```diff
@@ -0,0 +1,107 @@
+    ></script>
+  </head>
+
+  <body>
+    <div class="container">
+      <navbar class="navbar__container box-shadow mb-2">
+        <div class="navbar_nav">
+          <a href="#" class="link__nostyle fs-xl header__brand-name"> GK UI</a>
+
+          <button class="button">
+            <a href="#" class="link__nostyle">Home</a>
+          </button>
+
+          <button class="button">
+            <a href="#" class="link__nostyle">Installment</a>
+          </button>
+
+          <button class="button">
+            <a href="../pages/documentation.html" class="link__nostyle"
+              >Documnetation
+            </a>
+          </button>
+        </div>
+
+        <div class="navbar__git-icon">
+          <button class="button btn-icon">
+            <a
+              href="https://github.com/shubhamkumar648"
+              class="link__nostyle"
+              target="_blank"
+              ><i class="fab fa-github"></i
+            ></a>
+          </button>
+        </div>
+      </navbar>
+
+      <aside class="aside__container p-2">
+        <div class="aside__header">
+          <span class="fs-xl">Component</span>
+        </div>
+
+        <hr />
+
+        <div class="aside__content p-1">
+            <span> <a class="active link__nostyle" href="../component/avatar.html">Avatar</a></span>
+            <span><a class="active link__nostyle" href="../component/alert.html">Alert</a></span>
+            <span><a class="active link__nostyle" href="../component/badge.html">Badges</a></span>
+            <span> <a class="active link__nostyle" href="../component/button.html">button</a></span>
+            <span> <a class="active link__nostyle" href="../component/card.html">Card</a></span>
+            <span> <a class="active link__nostyle" href="../component/image.html">Image</a></span>
+            <span> <a class="active link__nostyle" href="../component/input.html">Input</a></span>
+            <span><a class="active link__nostyle" href="../component/textutilities.html">Text Utilities</a></span>
+            <span><a class="active link__nostyle" href="../component/navigation.html">Navigation</a></span>
+           <span> <a class="active link__nostyle" href="../component/snackbar.html">Snackbar</a></span>
+            <span><a class="active link__nostyle" href="../component/rating.html">Rating</a></span>
+        </div>
+      </aside>
+
+      <main class="main__container p-2">
+        <h3 class="">Text Utilities</h3>
+        <p class="ls-m mt-2 mb-2">
+          Text Utilitiesimproves site usability when text clarity and
+          readability concepts are applied.
+        </p>
+        <h6>Variant of Text: heading|center text|smalltext|gray text</h6>
+        <div class="text__container mb-2">
+          <h1 class="font-xxl">This is heading 1</h1>
+          <h2 class="font-xl">This is heading 2</h2>
+          <h3 class="font-l">This is headding 3</h3>
+          <h4 class="font-m">This is heading 4</h4>
```

Comment:
According to the preview, I think `heading 4` has more `font-weight` than `heading-5`.